habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hab3_hl_neural_ma.yaml

```diff
@@ -0,0 +1,31 @@
+name: "HierarchicalPolicy"
+obs_transforms:
+  add_virtual_keys:
+    virtual_keys:
+      "goal_to_agent_gps_compass": 2
+hierarchical_policy:
+  high_level_policy:
+    name: "NeuralHighLevelPolicy"
+    allowed_actions:
+      - nav
```

Comment:
What is the difference between 'nav', 'nav_to_obj', 'nav_to_goal' and 'nav_to_receptacle_by_name'? Why are they different between hab3_hl_neural and hab3_hl_neural_ma?

Reply:
It is confusing. `nav_to_obj`, `nav_to_goal`, and `nav_to_receptacle_by_name` are PDDL actions. `nav` is the skill (like a learned policy) that accomplishes all these PDDL actions. Unfortunately we need a separate `nav_to_obj` and `nav_to_goal` right now because I need to check the PDDL types that the robot is not holding the object it is trying to navigate to.